scania-digital-design-system/sdds
# fix(card): introduce correct colors for icon

**Describe pull-request**  
Introduced color variables for card icon in dark and light mode. The color values will be updated once the updates to Figma UI-kit have been finalised. This is a first idea, please let me know if there's a better way to do this.

**Solving issue**  
Fixes: [DTS-949](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-949)

**How to test**  
1. Go to Storybook link below
2. Check in Card -> Native and Web Component
3. Check that the arrow icon is grey-300 in light mode
4. Check that the arrow icon is grey-700 in dark mode

[DTS-949]: https://tegel.atlassian.net/browse/DTS-949?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
